### PR TITLE
Theming/cards

### PR DESCRIPTION
**Pull request by vyvil**
**PR to "Theme" branch.**

PR to adjust colors for "card-like" components (or using "card" color).
(excluding text inputs and modals for now)
PR does not introduce much changes - just adjust colors, removes unused props and adds components into storybook.

### CODE REVIEW

File: src/components/HeaderBlock/HeaderBlock.js [hunk 1]
```diff
@@ -146,18 +146,11 @@ const TextButton = styled.TouchableOpacity`
   padding: 2px 0;
   flex-direction: row;
   align-items: center;
-  ${props => props.withBackground
```
Comment (graszka22) [inline src/components/HeaderBlock/HeaderBlock.js:149]: delete also this prop set in `<TextButton />` below

File: src/components/HeaderBlock/HeaderBlock.stories.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      navigation={navigationMock}
+      noBack
+      centerItems={[{ title: 'Title' }]}
+      rightItems={[{
+        close: true,
+        onPress: () => {},
+      }]}
+    />
+  ))
+  .add('edge cases', () => (
```
Comment (graszka22) [inline src/components/HeaderBlock/HeaderBlock.stories.js:90]: I don't know if it's intended but on iphone 11 simulator it breaks the title like this:

Screenshot 2020-10-22 at 00 09 39

File: src/components/ListItem/SettingsItemCarded.js [hunk 3]
```diff
@@ -28,46 +27,31 @@ import { BaseText, MediumText } from 'components/Typography';
...
-  isActive?: boolean,
-  customIcon?: React.Node,
-  settingsIcon?: string,
-  settingsLabel?: string,
-  isLoading?: boolean,
-  onSettingsLoadingPress?: Function,
-  settingsIconSource?: string,
-  sidePaddingsForWidth?: number,
+  onPress?: () => void,
+  isActive?: boolean, //
```
Comment (graszka22) [inline src/components/ListItem/SettingsItemCarded.js:40]:
```suggestion
  isActive?: boolean,
```

File: src/components/ListItem/SettingsItemCarded.js [hunk 4]
```diff
@@ -28,46 +27,31 @@ import { BaseText, MediumText } from 'components/Typography';
...
   theme: Theme,
-  iconStyle?: Object,
-  isSwitching: boolean,
+  isLoading: boolean, //
```
Comment (graszka22) [inline src/components/ListItem/SettingsItemCarded.js:42]:
```suggestion
  isLoading: boolean,
```

File: src/navigation/appNavigation.js [hunk 5]
```diff
@@ -514,7 +522,7 @@ const tabNavigation = createBottomTabNavigator(
         shadowOffset: { width: 0, height: -2 },
         shadowOpacity: 0.05,
         shadowRadius: 2,
-        borderTopColor: 'transparent',
+        // borderTopColor: 'transparent',
```
Comment (graszka22) [inline src/navigation/appNavigation.js:525]:
```suggestion
```

File: src/components/QRCode/QRCodeWithTheme.js [hunk 6]
```diff
@@ -39,14 +38,13 @@ const QRCodeWithTheme = (props: Props) => {
...
   const colors = getThemeColors(theme);
-  const themeType = getThemeType(theme);
   return (
     <QRCode
       getRef={getRef}
       value={value}
       size={size}
-      color={themeType === DARK_THEME ? '#FFFFFF' : colors.text}
-      backgroundColor={colors.card}
+      color={getColorByThemeOutsideStyled(theme.current, { lightKey: 'basic010', darkCustom: '#FFFFFF' })}
```
Comment (graszka22) [inline src/components/QRCode/QRCodeWithTheme.js:46]: The qr code in dark mode has reversed black and white colors? Can't the dark mode be just normal?

Screenshot 2020-10-22 at 00 47 02

Screenshot 2020-10-22 at 00 46 11

Comment (vyvil) [reply src/components/QRCode/QRCodeWithTheme.js:46]: Yeah, it can :)
I'll update it

File: src/components/HeaderBlock/HeaderBlock.stories.js (hunk 2 above)
Comment (vyvil) [reply src/components/HeaderBlock/HeaderBlock.stories.js:90]: Thanks for spotting this! Fixed :)

**Review by graszka22 — APPROVED** [review]

**Pull request merged**